install_config/configuring_sdn.adoc

```diff
@@ -164,6 +164,181 @@ Check VNIDs by running:
+
+It is difficult to change the networking configuration of an
+{product-title} cluster after it has been deployed, but if absolutely
+necessary, it can be done.
+
+=== Expanding the Cluster IP Range By Adding Nodes
+
+The easiest reconfiguration involves changing the
+`*clusterNetworkCIDR*` to make room to add additional nodes, while
+leaving existing Nodes and Pods untouched.
+
+For example, with the default `*clusterNetworkCIDR*` value of
+*10.128.0.0/14* and the default `*hostSubnetLength*` of *9*, you can
+have 512 nodes, with Pod subnets from *10.128.0.0/23* to
+*10.131.254.0/23*. (Older versions of {product-title} had smaller
+defaults.) If you changed `*clusterNetworkCIDR*` to *10.128.0.0/13*,
+then you could have an additional 512 nodes, with Pod IPs going up to
+*10.135.255.255*.
+
+To do this, simply change the `*clusterNetworkCIDR*` value in the
+`*networkConfig*` stanza of the *_master-config.yaml_* file, and then
+restart the master; for single master clusters:
+
+----
+# systemctl restart atomic-openshift-master
+----
+
+For multi-master clusters:
+
+----
+# systemctl restart atomic-openshift-master-controllers
+----
+
+You will also need to restart the node service on the master and on
+each of the nodes, to make them update their IP routing information to
+take the larger cluster network into account:
+
+----
+# systemctl restart atomic-openshift-node
+----
+
+Note that this only works if you are changing `*clusterNetworkCIDR*`
+to a new value which is larger than the old value, and which includes
+the old value as a subset; if you try to change it to any other value,
+then when you restart the master, it will complain about the mismatch
+and refuse to start. You will need to use the procedure below in this
+case.
+
+=== Other Changes to the Cluster Network IP Range
+
+If you want to completely change `*clusterNetworkCIDR*` (eg, moving
+from *10.128.0.0/14* to *192.168.9.0/8*) or you want to change
+`*hostSubnetLength*` (to allow more or fewer pods per node), you will
+need to completely take down the cluster temporarily.
+
+. On every node and master, do:
++
+----
+# systemctl stop atomic-openshift-node
+# systemctl restart docker
+----
+
+. As a cluster admin user, run:
++
+----
+$ oc delete hostsubnets --all
+$ oc delete clusternetwork default
+----
+
+. Stop the masters. In a single-master cluster:
++
+----
+# systemctl stop atomic-openshift-master
+----
++
+For multi-master clusters:
++
+----
+# systemctl stop atomic-openshift-master-controllers
+# systemctl stop atomic-openshift-master-api
+----
+
+. Now edit the *_master-config.yaml_* file and make the desired changes.
+
+. Restart the masters:
++
+----
+# systemctl start atomic-openshift-master
+----
++
+or (multi-master):
++
+----
+# systemctl start atomic-openshift-master-controllers
+# systemctl start atomic-openshift-master-api
+----
+
+. Restart the node service on all of the nodes and masters:
++
+----
+# systemctl start atomic-openshift-node
```

Comment:
Do we want to restart ovs before this?  Or at least delete the bridge with our rules to get a clean slate?  I know we will work when we boot, but extra cruft would be left behind.